src/env.js

```diff
@@ -1,92 +1,35 @@
 // Replace with the URL where a Kafka REST service is listening
-var KAFKA_REST = "http://localhost:8082";// "https://kafka-rest-proxy.demo.landoop.com"
 
-var KAFKA_TOPIC_DELETE_COMMAND = "kafka-topics --zookeeper zookeeper-host:2181/confluent --delete --topic";
+var clusters =
+[
+  {
+  NAME:"prod",
+  KAFKA_REST : "localhost:8082",// "https://kafka-rest-proxy.demo.landoop.com"
+  KAFKA_REST_ENV : {
```

Comment:
`KAFKA_REST_ENV` maybe we don't need this to be nested now we simplified the env.js. could be `MAX_BYTES` blah blah